app/views/users/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<nav class="level">
+  <div class="level-left">
+    <h1 class="title">Users</h1>
+  </div>
+</nav>
+
+<%= paginate @users %>
+<table class="table is-bordered is-fullwidth">
+  <thead>
+    <tr>
+      <th><%= t('activerecord.attributes.user.name') %></th>
```

Comment:
DBのキーにあたるものは
https://railsguides.jp/i18n.html#active-recordモデルで翻訳を行なう
にあるようなメソッドを使うのがセオリーです。

ここの場合、`User.human_attribute_name(:name)`という感じで他も同様ですねー

Reply:
なるほど......！！！！
`i18n`のセオリーがわからず、全統一で`t`メソッド使った方が綺麗なのかなと思っていました。
DBカラム名に紐づくようなモデルのキーを表示させるときは`human_attribute_name`を使用して翻訳すべきなんですね、ありがとうございます 🙏

Reply:
余談ですが、
> i18nのセオリーがわからず、全統一でtメソッド使った方が綺麗なのかなと思っていました。

勿論内部では`t`メソッド使われていますねー。
https://github.com/rails/rails/blob/master/activemodel/lib/active_model/translation.rb#L44-L68
で、最後で
```ruby
I18n.translate(defaults.shift, **options)
```
してますね。(I18n.traslateのエイリアスメソッドがI18n.tです)
よって、`Model.human_attribute_name`はtメソッドのラッパーメソッドということになるかなーと思います。

こういう風にあるメソッド(引数が多い、とか、ややこしいとか、引数が長くなるとか)を別のメソッドで包んで単純化した呼び出しが出来るようにするのはよく使うテクニックだと思います!

Reply:
わ〜〜〜〜〜〜ありがとうございます 🙇 
自分1人ではソースコードの対応箇所まで探し当てることをしたことがなかったので感激しました！！！コードを読むことでラッパーメソッドになる、という意味もわかりました。初心者がRailsもRubyも便利に使えるのは意識しなくともこういうラッパーメソッドの恩恵を受けているからだなと思いました・・・！

Reply:
余談だらけですけど、こんな感じのコマンドで検索してヒットしたところ見にいったりしてますー。(もっといいやり方あるかもです) `rg`コマンドはripgrepといいまして、grepコマンドの代替で高速検索出来るものです。
https://github.com/BurntSushi/ripgrep
ですね。ご参考までにー。ripgrepについては過去に以下のQ&Aでも言及しましたー
https://bootcamp.fjord.jp/questions/688

```
maedana@manjaro ~/s/g/Y/Fjord_Rails_Task ((9c66dc2b…))> bundle show rails
[DEPRECATED] use `bundle info rails` instead of `bundle show rails`
The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x86-mswin32, x64-mingw32, java. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x86-mswin32 x64-mingw32 java`.
/home/maedana/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rails-6.0.3.2
maedana@manjaro ~/s/g/Y/Fjord_Rails_Task ((9c66dc2b…))> cd /home/maedana/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/
maedana@manjaro ~/.r/v/2/l/r/g/2/gems> rg 'def human_attirubute_name'
^C⏎                                                                                                                                                                                                                                                                             maedana@manjaro ~/.r/v/2/l/r/g/2/gems [SIGINT]> rg 'def human_attribute_name'
actionview-6.0.3.3/lib/action_view/helpers/tags/translator.rb
31:          def human_attribute_name

activemodel-6.0.3.2/lib/active_model/translation.rb
44:    def human_attribute_name(attribute, options = {})

actionview-6.0.3.2/lib/action_view/helpers/tags/translator.rb
31:          def human_attribute_name

activemodel-6.0.3.3/lib/active_model/translation.rb
44:    def human_attribute_name(attribute, options = {})
maedana@manjaro ~/.r/v/2/l/r/g/2/gems> rg 'def self.human_attribute_name'
activemodel-6.0.3.2/lib/active_model/errors.rb
37:  #     def self.human_attribute_name(attr, options = {})

activemodel-6.0.3.2/README.rdoc
133:      def self.human_attribute_name(attr, options = {})

activemodel-6.0.3.3/README.rdoc
133:      def self.human_attribute_name(attr, options = {})

activemodel-6.0.3.3/lib/active_model/errors.rb
37:  #     def self.human_attribute_name(attr, options = {})
```

Reply:
QAまでありがとうございます〜〜 😭 
FjordBootCampには有益な情報が流れすぎていて見落としがちなのですっごく助かりました 🙏 拝見します！